botCommands/points.js

```diff
@@ -170,6 +170,7 @@ const awardPoints = {
           return;
         }
         try {
+          const previousPoints = (await lookUpUser(user.id)).points;
           const pointsUser = await addPointsToUser(user.id, userId[1]);
```

Comment:
`pointsUser` has the `points`  property. Maybe we can use that to calculate `previousPoints` instead of making an API call. What do you think?